From 6f06147a5df6f5de0aa84412bc43db60afed67c9 Mon Sep 17 00:00:00 2001
From: Krzysztof Kozlowski <krzysztof.kozlowski@oss.qualcomm.com>
Date: Sat, 1 Aug 2026 21:54:41 +0200
Subject: [PATCH 10051/15333] regulator: dt-bindings: Correct white-space style

Correct a few white-space issues, like double space after '=' character,
which will be flagged by dt-check-style.  No functional changes.

Signed-off-by: Krzysztof Kozlowski <krzysztof.kozlowski@oss.qualcomm.com>
Link: https://patch.msgid.link/20260801195440.234183-2-krzysztof.kozlowski@oss.qualcomm.com
Signed-off-by: Mark Brown <broonie@kernel.org>
---
 .../bindings/regulator/nxp,pf8x00-regulator.yaml     | 12 ++++++------
 .../devicetree/bindings/regulator/ti,tps51632.yaml   |  2 +-
 2 files changed, 7 insertions(+), 7 deletions(-)

diff --git a/Documentation/devicetree/bindings/regulator/nxp,pf8x00-regulator.yaml b/Documentation/devicetree/bindings/regulator/nxp,pf8x00-regulator.yaml
index 894bdbca78a2..8ac77c1a9b9f 100644
--- a/Documentation/devicetree/bindings/regulator/nxp,pf8x00-regulator.yaml
+++ b/Documentation/devicetree/bindings/regulator/nxp,pf8x00-regulator.yaml
@@ -134,42 +134,42 @@ examples:
                     regulator-always-on;
                     regulator-boot-on;
                     regulator-max-microvolt = <1800000>;
-                    regulator-min-microvolt =  <400000>;
+                    regulator-min-microvolt = <400000>;
                 };
 
                 reg_buck2: buck2 {
                     regulator-always-on;
                     regulator-boot-on;
                     regulator-max-microvolt = <1800000>;
-                    regulator-min-microvolt =  <400000>;
+                    regulator-min-microvolt = <400000>;
                 };
 
                 reg_buck3: buck3 {
                     regulator-always-on;
                     regulator-boot-on;
                     regulator-max-microvolt = <1800000>;
-                    regulator-min-microvolt =  <400000>;
+                    regulator-min-microvolt = <400000>;
                 };
 
                 reg_buck4: buck4 {
                     regulator-always-on;
                     regulator-boot-on;
                     regulator-max-microvolt = <1800000>;
-                    regulator-min-microvolt =  <400000>;
+                    regulator-min-microvolt = <400000>;
                 };
 
                 reg_buck5: buck5 {
                     regulator-always-on;
                     regulator-boot-on;
                     regulator-max-microvolt = <1800000>;
-                    regulator-min-microvolt =  <400000>;
+                    regulator-min-microvolt = <400000>;
                 };
 
                 reg_buck6: buck6 {
                     regulator-always-on;
                     regulator-boot-on;
                     regulator-max-microvolt = <1800000>;
-                    regulator-min-microvolt =  <400000>;
+                    regulator-min-microvolt = <400000>;
                 };
 
                 reg_buck7: buck7 {
diff --git a/Documentation/devicetree/bindings/regulator/ti,tps51632.yaml b/Documentation/devicetree/bindings/regulator/ti,tps51632.yaml
index 67ac40e3ec00..130fa9796f57 100644
--- a/Documentation/devicetree/bindings/regulator/ti,tps51632.yaml
+++ b/Documentation/devicetree/bindings/regulator/ti,tps51632.yaml
@@ -44,7 +44,7 @@ examples:
 
         tps51632@43 {
             compatible = "ti,tps51632";
-            reg =  <0x43>;
+            reg = <0x43>;
             regulator-name = "tps51632-vout";
             regulator-min-microvolt = <500000>;
             regulator-max-microvolt = <1500000>;
-- 
2.55.0

